homeassistant/components/meater/sensor.py

```diff
@@ -142,26 +145,6 @@ def device_info(self):
             "manufacturer": "Apption Labs",
         }
 
-    @property
-    def available(self):
-        """Return if entity is available."""
-        if not self.coordinator.last_update_success:
-            return False
-
-        # See if the device was returned from the API. If not, it's offline
-        device = None
-
-        for dev in self.coordinator.data:
-            if dev.id == self.device_id:
-                device = dev
-
-        return device is not None
-
```

Comment:
I'm wondering if you don't want to overwrite this still. `CoordinatorEntity` only provides whether or not the update was successful, but you also have logic in there to see if the probe is still online. Taking this out will change the way the probes are treated when they're offline.  

That's totally up to you, just pointing out that this is different than the way it was before.

Reply:
I was thinking about this when I removed it. I wasn't sure it really added any benefit. This code specifically made the devices appear as "unavailable" when they're not connected to Meater Cloud. I know I personally don't like having devices listed as unavailable in my HA install, I almost use it as an impromptu check for stale devices. But that is just my personal preference. Thinking about it now, marking the probe device as unavailable probably is the correct behaviour.

Reply:
I guess my preference would be to keep them as unavailable when they're offline, just because it seems the most accurate way to describe the current state of the entities, but I agree I don't love seeing things marked as unavailable either. Maybe during the review process one of the devs will have a better idea on how to handle it.